content/en/apps/tutorials/sms-schedules.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Building SMS Forms"
+linkTitle: SMS Schedules
+weight: 6
+description: >
+  Building CHT application SMS forms
+relatedContent: >
+  apps/reference/app-settings/schedules
+
+---
+
+SMS schedules allow you to send reminder messages at predetermined times. SMS schedules can be triggered by [SMS forms]({{< ref "apps/tutorials/sms-forms" >}}) or App forms.
```

Comment:
It would be helpful to tell the reader why this feature can be useful.

Suggest moving this to the `key concepts` section
> SMS schedules can be triggered by [SMS forms]({{< ref "apps/tutorials/sms-forms" >}}) or App forms.

Reply:
Done.